src/com/flowers/kiev/entity/Flower.java

```diff
@@ -0,0 +1,16 @@
+package com.flowers.kiev.entity;
+
+public abstract class Flower {
+    private int price;
+
+    public Flower(int price) {
+        this.price = price;
+    }
+
+    public int getPrice() {
+        return price;
+    }
+
+    @Override
+    public abstract String toString();
```

Comment:
Good job. We make it abstract not to forget to implement this method in the class heir.